create-account.php

```diff
@@ -0,0 +1,6 @@
+<?php
+require_once "db.php";
+
+$db->query("insert into user (email, password, first_name, last_name) VALUES (?,?,?,?)", [$_POST["email"], $_POST["password"], $_POST["first-name"], $_POST["last-name"]]);
```

Comment:
This file isn't validating the password confirmation like register.php already had